### ISSUE TITLE
Up/Down mixed up

### ISSUE THREAD

**Issue by Robhorry** [loick111/homebridge-rfxcom-accessories#1]
**Describe Your Problem:**
Up and down is mixed up. RFXmngr Command works perfect.
Hombridge/Homekit Status is mixed up. Any Ideas?

**Logs:**
2021-11-03 21:25:49.288 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,01,01,01,01,01,01,00,00,00,00
2021-11-03 21:25:49.971 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,01,00
2021-11-03 21:25:49.973 [rfxcom] on /dev/ttyUSB0 - Response: Command message 01, ACK - transmit OK
2021-11-03 21:30:03.481 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,04,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:03.486 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,04,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:03.762 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,05,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:03.765 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,05,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:04.017 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,06,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:04.020 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,06,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:04.171 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,04,00
2021-11-03 21:30:04.171 [rfxcom] on /dev/ttyUSB0 - Response: Command message 04, ACK - transmit OK
2021-11-03 21:30:04.320 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,07,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:04.322 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,07,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:04.663 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,08,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:04.827 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,05,00
2021-11-03 21:30:04.828 [rfxcom] on /dev/ttyUSB0 - Response: Command message 05, ACK - transmit OK
2021-11-03 21:30:04.829 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,08,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:04.983 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,09,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:05.498 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,06,00
2021-11-03 21:30:05.499 [rfxcom] on /dev/ttyUSB0 - Response: Command message 06, ACK - transmit OK
2021-11-03 21:30:05.501 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,09,01,01,01,01,00,00,00,00,00
2021-11-03 21:30:06.170 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,07,00
2021-11-03 21:30:06.171 [rfxcom] on /dev/ttyUSB0 - Response: Command message 07, ACK - transmit OK
2021-11-03 21:30:06.842 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,08,00
2021-11-03 21:30:06.843 [rfxcom] on /dev/ttyUSB0 - Response: Command message 08, ACK - transmit OK
2021-11-03 21:30:07.514 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,09,00
2021-11-03 21:30:07.514 [rfxcom] on /dev/ttyUSB0 - Response: Command message 09, ACK - transmit OK
2021-11-03 21:30:14.017 [rfxcom] on /dev/ttyUSB0 - Queued  : 0C,1A,00,0A,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:14.020 [rfxcom] on /dev/ttyUSB0 - Sent    : 0C,1A,00,0A,01,01,01,01,03,00,00,00,00
2021-11-03 21:30:14.696 [rfxcom] on /dev/ttyUSB0 - Received: 04,02,01,0A,00
2021-11-03 21:30:14.696 [rfxcom] on /dev/ttyUSB0 - Response: Command message 0A, ACK - transmit OK
```

**Plugin Config:**

{
    "bridge": {
        "name": "",
        "username": "",
        "port": ,
        "pin": ""
    },
    "accessories": [
       
    ],
    "platforms": [
       {
            "name": "RFXCOM",
            "tty": "/dev/ttyUSB0",
            "debug": true,
            "discover": false,
            "devices": {
                "rfy": [
                    {
                        "name": "Buero_RFX",
                        "deviceId": "010101/1",
                        "openCloseDurationSeconds": "30"
                    }
                ]
            },
            "platform": "RFXCOMAccessories"
        }
    ]
}

**Screenshots:**
<!-- If applicable, add screenshots to help explain your problem. -->

**Environment:**

* **Plugin Version**: homebridge-rfxcom-accessories v2.3.6
* **Homebridge Version**: v1.3.5
* **Node.js Version**: v14.18.1
* **NPM Version**: v6.14.15
* **Operating System**: Raspbian /  hb-service

**Comment by loick111** [comment]
Hello,

Yes, I fixed the problem in pull request #2.
In fact in my use case it was more "horizontal" movements, so I reversed the direction.

Must be fixed now!

**Comment by loick111** [comment]
Fix is available on release v2.3.7
